tests/e2e/ansible/install_docker.yaml

```diff
@@ -71,6 +71,23 @@
       retries: 3
       delay: 10
   when: docker_exist.rc != 0 and ansible_distribution == "Ubuntu" and ansible_distribution_version == "22.04"
+- name: Handle docker installation on Ubuntu 24.04
+  block:
+    - name: Install docker packages
+      package:
+        name:
+          - docker.io
+          - docker-buildx
+        state: present
+      retries: 3
+      delay: 10
+    - name: Create the docker group
+      group:
+        name: docker
+        state: present
+      retries: 3
+      delay: 10
+  when: docker_exist.rc != 0 and ansible_distribution == "Ubuntu" and ansible_distribution_version == "24.04"
```

Comment:
Couldn't this be achieved by mydifying the 22.04 setup (`when: docker_exist.rc != 0 and ansible_distribution == "Ubuntu" and ansible_distribution_version in ("22.04", "24.04")`)

Reply:
Most likely. :-)
I will change that.